sentry_sdk/integrations/aws_lambda.py

```diff
@@ -25,6 +27,45 @@
 
     F = TypeVar("F", bound=Callable[..., Any])
 
+# Constants
+TIMEOUT_THRESHOLD_MILLIS = 1500  # Minimum time required to capture TimeoutError
```

Comment:
This sort of makes `AwsLambdaIntegration(check_timeout_error=True)` "maybe" work as intended, depending on external state that is not obvious when you initialize the integration. Do we really need this?